src/main/include/subsystems/Drivetrain.hpp

```diff
@@ -302,6 +310,12 @@ class Drivetrain : public ControlledSubsystemBase<7, 2, 5> {
     frc::ADIS16470_IMU m_imu;
     units::radian_t m_headingOffset = 0_rad;
 
+    frc::LinearQuadraticRegulator<2, 2> m_turningLQR{
+        kPlant, {50.0, 50.0}, {12.0, 12.0}, Constants::kControllerPeriod};
+
+    frc::LinearPlantInversionFeedforward<2, 2> m_turningFF{
+        kPlant, Constants::kControllerPeriod};
```

Comment:
This uses the wrong plant. You want the 3-state one.

Reply:
Yeah, so I can just plug in the function straight into it, instead of making a separate plant variable?

Reply:
That would work.